runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java

```diff
@@ -113,7 +113,6 @@ public boolean isMutation(UUID stream) {
         // Base Messages
         NOP(0, LogEntry.class),
         SMR(1, SMREntry.class),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SMREntry"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)